src/utility/appConstants.js

```diff
@@ -0,0 +1,7 @@
+export const APP_MESSAGES = {
+  requestTimedOut: 'Request timed out',
+};
+
+export const apiTimeOutLimit = 5000;
+export const baseURL = 'https://fakestoreapi.com';
+export const defaultCurrencyQuery = 'USD_INR';
```

Comment:
Move constants to `.env` file.

Reply:
Also, we are already using JSON data as the data source. Why do we need `https://fakestoreapi.com`?

Reply:
we don't have the data yet.